app/cli.py

```diff
@@ -155,6 +157,29 @@ def apikey():
         """apikey commands"""
         pass
 
+    @app.cli.command()
+    def check_bad_url():
+        """check for expired resource url"""
+        resources = Resource.query.all()
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) \
+                           AppleWebKit/537.36 (KHTML, like Gecko) \
+                           Chrome/79.0.3945.117 Mobile Safari/537.36'
+        }
+        processes = []
+
+        def print_bad_url(resource):
```

Comment:
When doing multithreaded work, printing your result isn't the most ideal outcome. Additionally making a new serialization format instead of using something that can be fed into the database to delete sounds like an antipattern. 

Instead of printing or not printing based on a bad result, you want return a boolean.

```python
# suggested name, pick whatever you want that doesn't involve printing. 
def filter_expired(resource):
       # check for bad url, return boolean
```

Reply:
fixed